support/syncLinkedPackageVersions.ts

```diff
@@ -0,0 +1,98 @@
+(async function(): Promise<void> {
```

Comment:
Can you rename this file to express what it does?

Reply:
Syncing the versions of linked packages is what this file does. Did you have some other name in mind?

Reply:
Oh, my bad. I misread the filename as `simLinkedPackageVersions`. 😅